syntax = "proto3";

package Asr;

service AsrService{
    rpc processAudio(stream AudioStream) returns (stream TextStream);
}
message AudioStream {
    bytes audio = 1;
    int32 id = 2;
}

message TextStream {
    string text = 1;
    int32 id = 2;
}
